content/scalability/docs/control-plane.md

```diff
@@ -286,10 +286,13 @@ Or list all pods that are running with:
 /api/v1/pods?fieldSelector=status.phase=Running&limit=500
 ```
 
-Another option to limit listed objects is to use [`resourceVersions` which you can read about in the Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions). Without a `resourceVersion` argument you will receive the most recent version available which requires an etcd quorum read which is the most expensive and slowest read for the database. The resourceVersion depends on what resources you are trying to query and can be found in the `metadata.resourseVersion` field.
+Another option to limit watch calls or listed objects is to use [`resourceVersions` which you can read about in the Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-versions). Without a `resourceVersion` argument you will receive the most recent version available which requires an etcd quorum read which is the most expensive and slowest read for the database. The resourceVersion depends on what resources you are trying to query and can be found in the `metadata.resourseVersion` field. This is also recommended in case of using watch calls and not just list calls
 
 There is a special `resourceVersion=0` available that will return results from the API server cache. This can reduce etcd load but it does not support pagination.
 
+```
+/api/v1/namespaces/default/pods?watch=true&resourceVersion=0
```

Comment:
I'd suggest removing these 3 lines (line 293 - 295) since some users may keeps using `resourceVersion=0` all the time which is not great either.
We should add something like the following after line 298:

It's recommended to use watch with a `resourceVersion` set to be the most recent known value received from its preceding list or watch. This is handled automatically in client-go. But it's suggested to double check it if you are using a k8s client in other languages.
```
/api/v1/namespaces/default/pods?watch=true&resourceVersion=362812295
```